code/network/multiutil.cpp

```diff
@@ -848,10 +848,8 @@ void multi_make_player_ai( object *pobj )
 		return;
 
     pobj->flags.remove(Object::Object_Flags::Player_ship);
-    auto flags = pobj->flags;
-    flags.set(Object::Object_Flags::Could_be_player);
-    flags.remove(Object::Object_Flags::Invulnerable);// Newly respawned players will still be invulnerable
-	obj_set_flags( pobj, flags );
+    pobj->flags.set(Object::Object_Flags::Could_be_player);
+	obj_set_flags( pobj, pobj->flags - Object::Object_Flags::Invulnerable);
```

Comment:
These two lines are not equivalent to the old code; I'm fairly certain that it should be `obj_set_flags( pobj, pobj->flags + Object::Object_Flags::Could_be_player - Object::Object_Flags::Invulnerable)` instead; If that doesn't work for some reason (and I don't see why it shouldn't), `Could_be_player` is the one that needs `obj_set_flags()` called for it.